src/wp-admin/about.php

```diff
@@ -7,14 +7,14 @@
  */
 
 /** ClassicPress Administration Bootstrap */
-require_once( dirname( __FILE__ ) . '/admin.php' );
+require_once dirname( __FILE__ ) . '/admin.php';
 
 wp_enqueue_script( 'underscore' );
 
 /* translators: Page title of the About ClassicPress page in the admin. */
 $title = _x( 'About', 'page title' );
 
-include( ABSPATH . 'wp-admin/admin-header.php' );
```

Comment:
`include()` changed to `require()`.

Reply:
As per review meeting, this is fine to keep the change